src/components/ui/ResultCount.js

```diff
@@ -0,0 +1,35 @@
+// @flow
+
+import React from 'react'
+import isNumber from 'lodash.isnumber'
+
+const cmz = require('cmz')
+
+const cx = {
+  result: cmz(`
+    font-size: 1.0625rem;
+  `),
+
+  resultNumber: cmz(`
+    font-weight: 600;
+    margin-right: 5px;
+  `)
+}
+
+type Props = {
+  items?: number
+}
+
+const ResultCount = ({ items }: Props) => {
+  const countItems = isNumber(items) ? items : 0
```

Comment:
I suggest to slightly change the order of words in the name, because when a variable starts with a verb, it naturally means an action rather than a container for a variable.
```suggestion
  const itemsCount = isNumber(items) ? items : 0
```